canvas-peer-grade-calculator/.env.sample

```diff
@@ -0,0 +1,60 @@
+# This `.env.sample` is based on
+# `longhornopen/canvas-peer-grade-calculator/web/.env.example`.
+
+# Name of the app.  Default is "laravel".
+APP_NAME="Peer Grading"
+# The environment the app is running in.  Supported: 'local', 'development',
+# or 'production'.
+APP_ENV=local
+# The app's encryption key.  Generate one with `php artisan key:generate`
+# or use the online tool at: https://generate-random.org/laravel-key-generator
+# The key string starts with "base64:".
+APP_KEY=
+# Whether error stack traces should be shown.  Set to 'false' in production
+APP_DEBUG=true
+# The base URL that this app is being served from.
+APP_URL=http://localhost
+
+# The access ID and key of the Canvas API key.
+# Info: `longhornopen/canvas-peer-grade-calculator/web/README.md`.
+CANVAS_DEVELOPER_ACCESS_ID=
+CANVAS_DEVELOPER_KEY=
+# The base URL of the Canvas installation
+CANVAS_URL=https://myschool.instructure.com
+
+# Where log files are to be written.
+# Info: `longhornopen/canvas-peer-grade-calculator/web/config/logging.php`.
+LOG_CHANNEL=errorlog
+LOG_LEVEL=debug
+
+# Session settings.
+# Info: `longhornopen/canvas-peer-grade-calculator/web/config/session.php`.
+
+# (Optional) Where session data is stored.  Supported: "file" (default),
+# "cookie", "database", "apc", "memcached", "redis", "dynamodb", or "array".
+# "file" is OK for local development or a single VM, but other options are
+# more suitable for Docker or load-balanced setups.
+SESSION_DRIVER=database
+
+# (Optional) Number of minutes a session may remain idle before it expires.
+# Default: 120.
+SESSION_LIFETIME=120
+
+# (Optional) Store session info in DB.
+# Use with `SESSION_DRIVER=database`.
+DB_CONNECTION=mysql
+DB_HOST=db
```

Comment:
This also worked with the container name of `peer_grading_db` directly with Docker. I think it's fine the way you have it documented.

Reply:
I just tried it out and indeed it does work.   Thanks for the tip!